docs/guide/index.md

```diff
@@ -62,7 +62,7 @@ $ pnpm create vite
 ```
 
 ```bash [Bun]
-$ bunx create-vite
+$ bun create-vite
```

Comment:
It should be like this:

```suggestion
$ bun create vite
```

Would you change it in all other places?

And thanks for the PR! (don't worry about pinging maintainers when you create one, it is better to leave that out so anyone in the project will feel they can go and review it)

Reply:
All right, I am still getting used to open source development